test/src/655-tarball_ingest_special_files/main

```diff
@@ -0,0 +1,119 @@
+
```

Comment:
Here I'd avoid re-testing was was tested before. Restrict it only to the more complex files.